magefiles/magefile.go

```diff
@@ -309,7 +309,7 @@ func (ci CI) TestE2E() error {
 }
 
 func RunE2ETests() error {
-	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup && !release-pipelines")
+	labelFilter := utils.GetEnv("E2E_TEST_SUITE_LABEL", "!upgrade-create && !upgrade-verify && !upgrade-cleanup")
```

Comment:
do we need this change? I guess this was done only for local testing?

Reply:
It's for testing the changed code, otherwise the test cases in "release-pipelines" suite will not be ran. I'll restore it before the PR is merged.

Reply:
Restored it.